bin/beaker

```diff
@@ -3,7 +3,11 @@
 require 'rubygems' unless defined?(Gem)
 require 'beaker'
 
-Beaker::CLI.new.execute!
-puts "Beaker completed successfully, thanks."
+if execute_subcommand?(ARGV[0])
```

Comment:
Okay so, if I set a version number for Thor (as per your other comment), should we merge this PR, and I can work on BKR-1026 separately? (that is, if we create a separate ticket for updating the rake task?)